packages/sanes-chrome-extension/src/routes/signup/test/fillSignupForm.ts

```diff
@@ -1,11 +1,29 @@
 import TestUtils from 'react-dom/test-utils';
 import { randomString } from '../../../utils/test/random';
 import { SECOND_STEP_SIGNUP_ROUTE } from '../components/ShowPhraseForm';
-import { findRenderedDOMComponentWithId } from '../../../utils/test/reactElemFinder';
-import { SECURITY_HINT_STEP_SIGNUP_ROUTE } from '../components/SecurityHintForm';
+import {
+  findRenderedDOMComponentWithId,
+  findRenderedE2EComponentWithId,
+} from '../../../utils/test/reactElemFinder';
+import { SECURITY_HINT_STEP_SIGNUP_ROUTE, SECURITY_HINT } from '../components/SecurityHintForm';
 import { sleep } from '../../../utils/timer';
 import { getHintPhrase } from '../../../utils/localstorage/hint';
 import { PersonaManager } from '../../../logic/persona';
+import { Page } from 'puppeteer';
+import { ACCOUNT_NAME_FIELD, PASSWORD_FIELD, PASSWORD_CONFIRM_FIELD } from '../components/NewAccountForm';
+
+export const submitAccountFormE2E = async (
+  page: Page,
+  accountName: string,
+  password: string
+): Promise<void> => {
+  await page.type(`input[name="${ACCOUNT_NAME_FIELD}"]`, accountName);
+  await page.type(`input[name="${PASSWORD_FIELD}`, password);
+  await page.type(`input[name="${PASSWORD_CONFIRM_FIELD}`, password);
+
+  await page.click('button[type="submit"]');
```

Comment:
Good job here, I wanted to do submitting the form but I prefer your solution. 👌

Reply:
I decided to use this approach because we are testing UI behavior, so we need to check submit button too.